glightning/lightning.go

```diff
@@ -2194,6 +2194,27 @@ func (l *Lightning) DevForgetChannel(peerId string, force bool) (*ForgetChannelR
 	return &result, err
 }
 
+type CustomMessageRequest struct {
+	NodeId  string `json:"node_id"`
+	Message string `json:"msg"`
+}
+
+func (r *CustomMessageRequest) Name() string {
+	return "dev-sendcustommsg"
```

Comment:
This need to be changed to `sendcustommsg`, but I think there is no reason to specify it https://github.com/ElementsProject/lightning/pull/4650 :)